repair/row_level.cc

```diff
@@ -495,8 +495,9 @@ void repair_writer_impl::create_writer(lw_shared_ptr<repair_writer> w) {
     }
     replica::table& t = _db.local().find_column_family(_schema->id());
     rlogger.debug("repair_writer: keyspace={}, table={}, estimated_partitions={}", w->schema()->ks_name(), w->schema()->cf_name(), w->get_estimated_partitions());
+    service::frozen_topology_guard topo_guard; // FIXME: propagate
```

Comment:
By default,  we use repair to stream data for topology change. We need the topo_guard for repair  with higher priority than streaming.

Reply:
Yes.